x-pack/plugins/cloud_security_posture/common/utils/helpers.ts

```diff
@@ -19,3 +21,14 @@ export const extractErrorMessage = (e: unknown, defaultMessage = 'Unknown Error'
 
   return defaultMessage; // TODO: i18n
 };
+
+export const filterByPackagePolicy = ({
```

Comment:
This method does not actually filter anything, it creates a filter, so consider renaming to better reflect that

Reply:
will do